graylog2-web-interface/src/components/graylog/AccordionItem.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+import Panel from './Panel';
+
+type Props = {
+  children: React.ReactNode,
+  name: string,
+  id?: string,
+}
+
+const StyledPanel = styled(Panel)`
+  border: 0;
+  border-radius: 0;
+`;
+
+const PanelHeading = styled(Panel.Heading)`
+  padding: 0;
+  border-radius: 0;
+`;
+
+const PanelTitle = styled(Panel.Title)(({ theme }) => css`
+  font-size: ${theme.fonts.size.small};
+  color: ${theme.colors.global.link};
+  
+  > a {
+    padding: 3px 9px;
+    display: block;
+  }
+`);
+
+const PanelBody = styled(Panel.Body)(({ theme }) => css`
+  background-color: ${theme.colors.variant.lightest.default};
+  border-top-color: ${theme.colors.variant.light.default} !important;
```

Comment:
Why is `!important` needed on a styled component? Is it not the top hierarchy already? Or are other styles loaded later down the line?

I think it would be good to have styled components always have 1 level of priority higher than the rest of the style definitions.

This is not up to you to fix now, but I guess it's a good discussion to be had.

Reply:
I agree with you, but unfortunately Bootstrap CSS does not. Without writing some obnoxious amount of CSS to override a single border color, I opted for `!important`. 

`.eGCokf > .Panel__PanelHeading-niu1uf-0 + .panel-collapse > .panel-body`

Reply:
*shakes fist at bootstrap*

Then `!important` is a good solution